### PR TITLE
Merge restyling, replace dat, add loader and avatar

### PR DESCRIPTION
**Pull request by ninabreznik**

### CODE REVIEW

File: demo/node_modules/contracts-db/index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ function contractsDB (daturl) {
 
   var counter = 0
   const archive = Hyperdrive(daturl)
+  window.archive = archive
```
Comment (serapath) [inline demo/node_modules/contracts-db/index.js]: this one can be removed i think

File: demo/node_modules/contracts-db/index.js [hunk 2]
```diff
@@ -40,36 +41,32 @@ function contractsDB (daturl) {
...
   }
-  // retrieve all sources
+    // retrieve all sources
   function getAll (done) {
+    console.log('Retreiving data from the P2P database')
     archive.ready(() => {
       archive.readdir('.', (err, addresses) => {
         if (err) return done(err)
         if (addresses) {
+          counter = addresses.length
```
Comment (serapath) [inline demo/node_modules/contracts-db/index.js]: why did this method change so much?
inside `getContractsArr` is said `counter = counter + sourcesArr.length`
that seems to have changed in `getData` to `counter = addresses.length` - is that correct?

File: src/node_modules/copyToClipboard.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        try {
+            copiableContent = hash
+        } catch (e) {
+            console.log('copy failed: ', e.message)
+            return
+        }
+        if (copiableContent) {
+            copy(copiableContent)
+            // set new notify message
+            let notification = notify(message)
```
Comment (serapath) [inline src/node_modules/copyToClipboard.js]: Hm, I personally feel, that all this modal stuff should maybe be handled by the `notify` module, so that any module which requires the `notify` module, can just send a notification, but does not need to bother how it gets displayed

File: src/node_modules/header.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ module.exports = header
 function header () {
   return bel`<header class="${css.header}">
     <div class="${css.logo}" onclick=${home}>
-      <img src="/src/assets/images/logo-1.png" alt="smartcontract.codes">
+      <img src="./assets/images/logo-1.png" alt="smartcontract.codes">
```
Comment (serapath) [inline src/node_modules/header.js]: is this going to work?
i imagine:
* `https://smartcontract.codes/index.html` needs to load
* `https://smartcontract.codes/src/assets/images/logo-1.png`
but also:
* `https://ninabreznik.github.io/smartcontract.codes/index.html` needs to load
* `https://ninabreznik.github.io/smartcontract.codes/src/assets/images/logo-1.png`

so `/src/assets/images/logo-1.png` does not work for tha latter case, because it would try to load:
* `https://ninabreznik.github.io/src/assets/images/logo-1.png`

but i think the former case does also not work, because  `./assets/images/logo-1.png` would try to load:
* `https://ninabreznik.github.io/smartcontract.codes/assets/images/logo-1.png`

---
So i would assume the correct way to load things would be to use:
* `src="src/assets/images/logo-1.png"`

File: src/node_modules/makeCollectionArea.js [hunk 5]
```diff
@@ -5,10 +5,12 @@ const makeCard = require('makeCard')
 module.exports = makeCollectionArea
 
 function makeCollectionArea (contracts) {
-  const cards = contracts.length ?
-    contracts.map(contract => makeCard(contract))
-    : bel`<div class=${css.noResult}>No matches found</div>`
-  return bel`<div class=${css.collectionArea}>${cards}</div>`
+  if (contracts.length)  {
```
Comment (serapath) [inline src/node_modules/makeCollectionArea.js]: why was this change necessary?
Before it would display either:
1. a filled collectionArea
2. or an empty collectionArea

...i guess if it works its fine, i'm just curious what this change is about :-)

File: src/node_modules/notify.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+let bel = require('bel')
+let csjs = require('csjs-inject') 
+
+module.exports = notify
```
Comment (serapath) [inline src/node_modules/notify.js]: cool idea to make a separate module, but i think this module could basically also handle all the logic of where and how and how long to display notifications and what to do if multiple notifications are displayed quickly, etc...

File: src/node_modules/search.js [hunk 7]
```diff
@@ -33,9 +38,9 @@ function preventDefault (e) {
   const keyCode = e.keyCode
   if (keyCode === 13 && !e.shiftKey) e.preventDefault()
 }
-function trigger (e, notify, searchArea) {
+function trigger (e, ops, searchArea) {
```
Comment (serapath) [inline src/node_modules/search.js]: something in this module went badly wrong.
on one hand, the module has a parameter `notify` already, but on the other hand it is still using `ops` ...which is pretty weird.
Maybe this particular file can be changed back?

File: src/node_modules/copyToClipboard.js (hunk 3 above)
Comment (ninabreznik) [reply src/node_modules/copyToClipboard.js]: @serapath Can you please make an issue for this and @fionataeyang can then fix it in the next days

File: src/node_modules/header.js (hunk 4 above)
Comment (ninabreznik) [reply src/node_modules/header.js]: Will try

File: src/node_modules/makeCollectionArea.js (hunk 5 above)
Comment (ninabreznik) [reply src/node_modules/makeCollectionArea.js]: To display No matches found

File: src/node_modules/notify.js (hunk 6 above)
Comment (ninabreznik) [reply src/node_modules/notify.js]: I agree, this is @fionataeyang 's firs module, maybe you can create another issue and she can also adapt this

File: src/node_modules/search.js (hunk 7 above)
Comment (ninabreznik) [reply src/node_modules/search.js]: We cna check with @fionataeyang . I didn't do any changes in this file, but not sure if she did or not
